### PR TITLE
this ic was hard

### PR DESCRIPTION
**Pull request by IcelandicGambit**
Rough.

### CODE REVIEW

**Comment by IcelandicGambit** [comment]
Sorry, I forgot to add my name in the title. Won't happen again.

File: lib/apartment.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+              :bathrooms,
+              :bedrooms,
+              :renter
+
+  def initialize(info)
+    @number = info[:number]
+    @monthly_rent = info[:monthly_rent]
+    @bathrooms = info[:bathrooms]
+    @bedrooms = info[:bedrooms]
+    @renter = nil
```
Comment (mikedao) [inline lib/apartment.rb:13]: Unnecessary

File: lib/building.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  def add_unit(unit)
+    units << unit
+  end
+
+  def renters
+    @units.each do |unit|  #refactor, map would probably be better?
+      @renters << unit.renter.name
+    end
+    @renters
```
Comment (mikedao) [inline lib/building.rb:20]: This does not handle empty units

**Review by mikedao — COMMENTED** [review]
What you have is good, but we would have liked to have seen you get further.

Score: 2.0

**Pull request closed without merging**